cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-logging.adoc

```diff
@@ -0,0 +1,235 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-logging"]
+= Tutorial: Logging
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-logging
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-05-01
+
+There are various methods to view your logs in {product-rosa} (ROSA). Use the following procedures to forward the logs to AWS CloudWatch, view the logs directly through the pod by using `oc logs`, and use LokiStack to search the logs.
+
+[NOTE]
+====
+ROSA is not preconfigured with a logging solution.
+====
+
+The cluster logging components are based on Fluentd and Elasticsearch (ES). Fluentd is deployed to each node in the cluster. It collects application logs and either writes them to ES or forwards them to CloudWatch.
```

Comment:
I think we need to look at the logging stack docs for OCP as they have recently migrated off a few of these tools.  I think LokiStack is now used instead of elastic and Vector instead of fluentd.  I also think Kibana may no longer be used. Can you get someone from the logs/observability team to help make sure the intro is correct?